### PR TITLE
[TASK] Reformat code to PSR-2 + normalize file header comment

### PR DESCRIPTION
**Pull request by dfeyer**

### CODE REVIEW

**Comment by dfeyer** [comment]
:+1: 

When this one is merged we can configure StyleCI

File: Classes/Flowpack/ElasticSearch/Domain/Model/Index.php [hunk 1]
```diff
@@ -18,211 +18,224 @@
...
-		}
-		return $settings;
-	}
+class Index
+{
+    /**
+     * @var array
+     * @see http://www.elasticsearch.org/guide/en/elasticsearch/reference/current/indices-update-settings.html
+     */
+    protected $updatableSettings= array(
```
Comment (johannessteu) [inline Classes/Flowpack/ElasticSearch/Domain/Model/Index.php]: Missing space ?!

**Comment by johannessteu** [comment]
Beside one missing space looks good for me :+1:

File: Classes/Flowpack/ElasticSearch/Domain/Model/Index.php (hunk 1 above)
Comment (dfeyer) [inline Classes/Flowpack/ElasticSearch/Domain/Model/Index.php]: Good catch, the index was missing before ;) but I fix this force push and merge this one

**Pull request merged**